### ISSUE TITLE
How to visualize the test result ?

### ISSUE THREAD

**Issue by ajinkya933** [ZhihengCV/Bayesian-Crowd-Counting#14]
I have downloaded your pre-trained model and im running test.py :

```python3 test.py --data-dir processed_data --save-dir logs```
In output I get
```
('img_0001',) -317.4151611328125 975 1292.4151611328125
('img_0002',) 234.6561279296875 923 688.3438720703125
```

My question is where is the visualization (heatmap) image saved ?

**Comment by rongliangzi** [comment]
Hi! Here is my codes to visualize the estimated density map:
`dm = outputs.squeeze().detach().cpu().numpy()
plt.imshow(dm, cmap=CM.jet)
plt.savefig(save_path)`

**Comment by ZhihengCV** [comment]
The visualization code is right.

**Comment by d6h2p** [comment]
你好，这样可视化是否正确？
dm = outputs.squeeze().detach().cpu().numpy()

dm_normalized = dm / np.max(dm)

plt.imshow(dm_normalized, cmap=cm.jet, vmin=0, vmax=1)
 
i += 1
 
plt.savefig("./image/{}.png".format(i))

**Comment by LOVEISI** [comment]
> 你好！这是我可视化估计密度图的代码：
> 
> ```
> dm = outputs.squeeze().detach().cpu().numpy()
> plt.imshow(dm, cmap=CM.jet)
> plt.savefig(save_path)
> ```

你好！请问这段代码是加在test.py后面吗
